### PR TITLE
fix merge problems 

### PR DESCRIPTION
**Pull request by yuukiiwa**
hi @cying111, I have re-forked the nf-core/nanoseq, which got some updates last week. I was trying to fix the merge problem, but it didn't quite work before I re-forked it. Now, the yuukiiwa/nanoseq directory can merge with nf-core/nanoseq.

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -1,153 +1,8 @@
 # nf-core/nanoseq: Usage
```
Comment (cying111) [inline docs/usage.md:1]: it seems all usage.md are cleaned, is this correct?

File: nextflow.config [hunk 2]
```diff
@@ -9,6 +9,7 @@
 params {
 
   // Options: Mandatory
+  input = './samplesheet.csv'
```
Comment (cying111) [inline nextflow.config:12]: what does this mean? does this mean that the input has to be provided with this name?  would it conflict with the input = False

File: docs/usage.md (hunk 1 above)
Comment (yuukiiwa) [reply docs/usage.md:1]: I think he got all the flags off from the usage to fix merge conflicts.

File: nextflow.config (hunk 2 above)
Comment (yuukiiwa) [reply nextflow.config:12]: I have removed the `input = false`, and now the nextflow.config follows the one on nf-core/nanoseq.

**Review by cying111 — APPROVED** [review]

**Pull request merged**